README.md

```diff
@@ -0,0 +1,94 @@
+# NETGEAR Monitoring Plugin for Icinga
+
+## Table of Contents
+- [About](#about)  
+- [License](#license)  
+- [Documentation](#documentation)  
+- [Support](#support)  
+- [Requirements](#requirements)  
+- [Thanks](#thanks)  
+- [Contributing](#contributing)
+---
+
+## About
+The NETGEAR Monitoring Plugin is a lightweight Go-based command-line tool designed to collect and report hardware and network metrics from NETGEAR devices via their API.
+It provides output using go-check, allowing easy integration into Icinga 2 checks.
+**Important:** This plugin is intended **only for use with NETGEAR AV Line devices**. Use with other devices is not supported and may produce incorrect results.
```

Comment:
I would highlight this as a warning or notice like this if it's that important to inform the users about it:
```
> [!WARNING]
>
> This plugin is intended **only for use with NETGEAR AV Line devices**. Use with other devices is
> not supported and may produce incorrect results.
```

GitHub will nicely render this once merged to main branch.